osbs/repo_utils.py

```diff
@@ -134,6 +134,15 @@ def __init__(self, dir_path='', depth=None, git_uri=None, git_branch=None, git_r
             possible_filename_typos=REPO_CONTENT_SETS_FILE_POSSIBLE_TYPOS
         )
 
+        if 'autorebuild' in self.container:
+            logger.warning('Autorebuilds are deprecated in OSBS 2.0, this config is be ignored')
+            del self.container['autorebuild']
+
+        if 'image_build_method' in self.container:
+            logger.warning("'image_build_method' config is deprecated in OSBS 2.0, this config "
```

Comment:
Maybe these should be `logger.user_warning` instead? I don't recall to what extext osbs-client supports our custom user warnings mechanism, but if it does, that would be more appropriate for deprecated properties

Reply:
switched to `user_warning`